echopype_sonarnetcdf4_compliance.md

```diff
@@ -28,6 +30,8 @@ Disruptive changes are preced by the prefixed **(D)**
   - Add missing core CF attributes (feature_type, cdm, etc), where appropriate
   - Add some useful attributes from ACDD, specially ones involving spatial and temporay bounding boxes; roles; etc; in top-level group? Will improve discoverability.
   - See AcMeta to see if we can define an initial subset of it that ideally follows the ACDD attributes we'd be interested in
+  - > (from WJ): Do you think these additions would potentially make their way into later versions of the convention?
+
```

Comment:
Good questions. I don't *think* they're in the immediate plans. But I think doing this would be both non-disruptive, would not break our convention adherence, and would actually be of direct use to us and anyone using datasets generated by echopype. Part of my goal in adding ACDD attributes is to serve as a concrete demonstration to bring back to SONAR-netCDF4 discussions, later on.